src/AssessSpatialKernelVisitor.cc

```diff
@@ -59,13 +59,13 @@ namespace detail {
         afwMath::CandidateVisitor(),
         _spatialKernel(spatialKernel),
         _spatialBackground(spatialBackground),
-        _ps(ps),
-        _imstats(ImageStatistics<PixelT>(_ps)),
+        _ps(ps.deepCopy()),
+        _imstats(ImageStatistics<PixelT>(ps)),
         _nGood(0),
         _nRejected(0),
         _nProcessed(0),
-        _useCoreStats(_ps.getAsBool("useCoreStats")),
-        _coreRadius(_ps.getAsInt("candidateCoreRadius"))
+        _useCoreStats(ps.getAsBool("useCoreStart")),
```

Comment:
Is this a new typo or fixing an old one?

Reply:
I have no idea how I introduced it in my edits but the tests told me pretty quickly that I had broken it.